src/main/scala/com/typesafe/jse/Trireme.scala

```diff
@@ -144,6 +163,12 @@ private[jse] class TriremeShell(
       })
   }
 
+  override def postStop() = {
+    // The script pool is a cached thread pool so it should shut itself down, but it's better to clean up immediately,
+    // and this means that our tests work.
+    nodeEnv.getScriptPool.shutdown()
+    nodeEnv.getScriptPool.awaitTermination(AwaitTerminationTimeout.toMillis, TimeUnit.MILLISECONDS)
```

Comment:
Is it necessary to block and wait? 

If so then perhaps it should be called from within a `blocking` function?